mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -8,86 +8,101 @@
-                                                    variableList=missingvars,
-                                                    yearOffset=1850)
+                                                    variableList=missingvars)
 
     def test_iselvals(self):
         fileName = str(self.datadir.join('example_jan.nc'))
+        calendar = 'gregorian_noleap'
+        simulationStartTime = '0001-01-01'
         timestr = 'time_avg_daysSinceStartOfSim'
         variableList = \
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
         iselvals = {'nVertLevels': slice(0, 3)}
-        ds = mpas_xarray.open_multifile_dataset(fileNames=fileName,
-                                                timeVariableName=timestr,
-                                                variableList=variableList,
-                                                iselValues=iselvals,
-                                                yearOffset=1850)
+        ds = mpas_xarray.open_multifile_dataset(
+            fileNames=fileName,
+            calendar=calendar,
+            simulationStartTime=simulationStartTime,
+            timeVariableName=timestr,
+            variableList=variableList,
+            iselValues=iselvals)
 
         self.assertEqual(sorted(ds.data_vars.keys()), sorted(variableList))
         self.assertEqual(ds[variableList[0]].shape, (1, 7, 3))
         self.assertEqual(ds['refBottomDepth'].shape, (3,))
         self.assertApproxEqual(ds['refBottomDepth'][-1],
                                4.882000207901)
-        date = pd.Timestamp(ds.Time.values[0])
-        # round to nearest second
-        date = pd.Timestamp(long(round(date.value, -9)))
-        self.assertEqual(date, pd.Timestamp('1855-01-13 12:24:14'))
+
+        self.assertEqual(days_to_datetime(days=ds.Time.values[0],
+                                          referenceDate='0001-01-01',
+                                          calendar=calendar),
+                         string_to_datetime('0005-01-14 12:24:14'))
 
     def test_no_units(self):
         fileName = str(self.datadir.join('example_no_units_jan.nc'))
+        calendar = 'gregorian_noleap'
+        simulationStartTime = '0001-01-01'
         timestr = 'time_avg_daysSinceStartOfSim'
         variableList = \
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
-        ds = mpas_xarray.open_multifile_dataset(fileNames=fileName,
-                                                timeVariableName=timestr,
-                                                variableList=variableList,
-                                                yearOffset=1850)
+        ds = mpas_xarray.open_multifile_dataset(
+            fileNames=fileName,
+            calendar=calendar,
+            simulationStartTime=simulationStartTime,
+            timeVariableName=timestr,
+            variableList=variableList)
```

Comment:
No, this is a correct PEP8 indentation convention, made necessary because `simulationStartTime=simulationStartTime` is too long with the previous indentation convention.  There is not a good way of indenting to match with open parenthesis without having the line be too long.